content/blog/2019/istio1.1_perf/index.md

```diff
@@ -0,0 +1,61 @@
+---
+title: Architecting Istio 1.1 for Performance
+description: An overview of Istio 1.1 performance.
+publishdate: 2019-03-18
+subtitle: An overview of Istio 1.1 performance improvements
+attribution: Surya V Duggirala (IBM), Mandar Jog (Google), Jose Nativio (IBM)
+keywords: [kubernetes,docker,container,performance,mesh]
```

Comment:
Replace these keywords with:

performance,scalability,scale,benchmarks

This will create a much more relevant See Also section